src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -80,6 +82,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddAssignmentCommand.COMMAND_WORD:
             return new AddAssignmentParser().parse(arguments);
 
+        case AddTaskCommand.COMMAND_WORD:
```

Comment:
We should first merge the PR which abstracts out the parsing of contact list, module list and todo list into individual classes. Following the merge of the above mentioned PR, please handle the parsing of Task related commands in the TodolistParser class.

Reply:
Okay, noted